modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase

```diff
@@ -3173,6 +3173,40 @@ definition {
 			value1 = "An unexpected error occurred while uploading your file.");
 	}
 
+	@description = "LPS-176823 - Verify that the Output Field on Formula Field is required"
+	@priority = 4
+	test OutputFieldIsRequiredOnFormulaField {
+		property portal.acceptance = "true";
```

Comment:
This does not need to be on acceptance so you can remove this line. Acceptance is usually for priority 5 tests or if it's specifically stated